addons/calendar/static/src/views/attendee_calendar/attendee_calendar_controller.xml

```diff
@@ -6,5 +6,42 @@
                 <button class="btn btn-primary o-calendar-button-new me-1" t-on-click="onClickAddButton">New</button>
             </t>
         </xpath>
+        <!-- Add header div to be filled with synchronization buttons in synchronization modules. -->
+        <xpath expr="//Layout//div[hasclass('o_calendar_container')]//h5[hasclass('d-inline-flex')]" position="after">
+            <div id="header_synchronization_settings" class="mx-2 ms-lg-auto">
+                <t t-if="(model.credentialStatus.google_calendar or model.credentialStatus.microsoft_calendar)
+                and (!model.syncStatus.google_calendar or ['sync_stopped', 'missing_credentials'].includes(model.syncStatus.google_calendar))
+                and (!model.syncStatus.microsoft_calendar or ['sync_stopped', 'missing_credentials'].includes(model.syncStatus.microsoft_calendar))">
+                    <h5 id="synchronize_with" class="d-inline-flex" >Synchronize with</h5>
+                </t>
+                <t t-elif="(model.googleCredentialsSet and model.syncStatus.google_calendar == 'sync_active')
+                or (model.microsoftCredentialsSet and model.syncStatus.microsoft_calendar == 'sync_active')">
+                    <div id="google_calendar_sync" class="o_calendar_sync mx-1 d-inline-flex">
+                        <button class="btn text-nowrap"
+                        t-on-click="model.syncStatus.google_calendar == 'sync_active' ? onStopGoogleSynchronization : onStopMicrosoftSynchronization">
+                            <div>
+                                <i id="check_symbol" class='fa fa-check px-1 o_text_green o_calendar_check'/>
+                                <i id="stop_symbol" class='fa fa-square px-1 o_text_red o_calendar_stop'/>
+                                <span id="check_text" class='o_calendar_check'>
+                                    <t t-if="model.syncStatus.google_calendar == 'sync_active'">Google</t>
+                                    <t t-else="">Outlook</t>
+                                </span>
```

Comment:
does it make sense in module calendar ?  Google / outlook

Reply:
google depend of calendar and microsoft depend of calendar

So if you set a method that google/microsoft override to add the status... it should works no ?  The same that you did with `check_synchronization_status`...

If I'm a partner and need to add the synchro to another service, how can I integrate to it ? I have no hook to specify my status... 

> since we can show only one label at the header (Active for X or Paused for Y)

Not sure why, irl you will have only one enabled probably, so always one button... and in worst case you like 'usine a gaz' and play with 3 calendars..., what the problem to have 2 buttons... 

Btw, now it is merged... it'll wait a future refactoring... 
Ty for your dev